enterprise/server/remote_execution/container/container.go

```diff
@@ -50,7 +50,7 @@ type CommandContainer interface {
 	//
 	// It is approximately the same as calling PullImageIfNecessary, Create,
 	// Exec, then Remove.
-	Run(ctx context.Context, command *repb.Command, workingDir string, creds PullCredentials) *interfaces.CommandResult
+	Run(ctx context.Context, task *repb.ExecutionTask, workingDir string, creds PullCredentials, fsLayout *FileSystemLayout) *interfaces.CommandResult
```

Comment:
I guess `task` and `fsLayout` are still being passed in because firecracker will need them -- but in the interest of simplifying the `CommandContainer` interface, I think it would be a bit cleaner to also do the firecracker-specific FS stuff inside `PrepareForTask` in runner.go as well. Kinda like

```
if ctr, ok := r.CommandContainer.(*FireCrackerContainer); ok {
  ctr.SetupCASFSInGuestVM(task, fsLayout)
} else {
  workspaceDir = makeCASFS()
}
```

Reply:
Done.